pkg/stellar/stellar.go

```diff
@@ -131,11 +133,23 @@ func (w *Wallet) CreateAccount() (string, string, error) {
 		return "", "", errors.Wrap(err, "failed to get source account")
 	}
 
-	err = w.activateEscrowAccount(newKp, sourceAccount, client)
-	if err != nil {
-		return "", "", errors.Wrapf(err, "failed to activate escrow account %s", newKp.Address())
+	activateEscrowBackoff := func() error {
+		log.Info().Msg("Trying to activate escrow account")
+		return w.activateEscrowAccount(newKp, sourceAccount, client)
 	}
 
+	bo := backoff.NewExponentialBackOff()
+	bo.MaxElapsedTime = time.Minute * 5 // retry for 5 minutes
+	bo.MaxInterval = time.Second * 30
+	backoff.RetryNotify(activateEscrowBackoff, bo, func(err error, d time.Duration) {
```

Comment:
This can still return an error when it fails permanently (when it give up retrying). This error then need to be checked  and returned to use if it happens.
```go
err := backof.RetryNotify()

if err != nil {
   return "", "", errors.Wrapf(err, "failed to activate escrow account %s", newKp.Address())
}
```

Reply:
otherwise, the process will continue even if activation was failed.